#!/usr/bin/env python
PACKAGE = "viual_grab"

from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator() 

gen.add( "nav_ptp_time", int_t, 0,"point to point navigation time",18, 0, 300)
gen.add( "navigation_ptp_timeout", int_t, 0,"point to point navigation timeout",60, 0, 300)

exit(gen.generate(PACKAGE, "viual_grab", "ExecuteMoveAndGrab"))
